Doc/library/pdb.rst

```diff
@@ -580,17 +585,9 @@ can be overridden by the local file.
 .. pdbcommand:: interact
 
    Start an interactive interpreter (using the :mod:`code` module) whose global
-   namespace contains all the (global and local) names found in the current
-   scope. Use ``exit()`` or ``quit()`` to exit the interpreter and return to
-   the debugger.
-
-   .. note::
-
-      Because interact creates a new global namespace with the current global
-      and local namespace for execution, assignment to variables will not
-      affect the original namespaces.
-      However, modification to the mutable objects will be reflected in the
-      original namespaces.
```

Comment:
This change is not correct, and I need to revert it. (I missed the `**` expansion that makes the copies when reading the code)

Reply:
This note has been restored (in a reworded form)